src/main/java/am/ik/yavi/core/NestedConstraintCondition.java

```diff
@@ -31,6 +31,10 @@ public NestedConstraintCondition(Function<T, N> nested,
 	@Override
 	public boolean test(T t, ConstraintGroup constraintGroup) {
 		final N n = this.nested.apply(t);
+		if (n == null) {
+			return false;
```

Comment:
Let me think carefully about whether the value changes when using `nest` and when using `nestIfPresent`.

Reply:
Seems OK